src/app/components/Pages/Player/stats/index.tsx

```diff
@@ -31,24 +32,29 @@ export default function StatsTab(props) {
                 {props.rank ? (
                     <>
                         <RankOverview {...props} />
-                        <div className="playermodule__divider" />
                         <SeasonOverview {...props} />
-                        <div className="playermodule__divider" />
                     </>
                 ) : null}
                 {props.aliases && props.aliases.length > 1 ? <Aliases {...props} /> : null}
             </div>
             <div className="statstab__content">
-                <div className="statstab__center">
-                    {props.stats ? <TimeSwitch {...props} /> : null}
-                    {props.stats ? <StatsGeneral {...props} stats={stats} /> : null}
-                    <StatsRankings {...props} rank={rank} />
-                    <div className="playermodule__divider" />
-                    {props.season === -1 && props.stats ? <StatsCharts {...props} /> : null}
+                {props.stats ? <StatsGeneral {...props} stats={stats} /> : null}
+                <div className="statstab__row">
+                    <div className="statstab__col">{props.stats ? <StatsCasual {...props} stats={stats} /> : null}</div>
+                    <div className="statstab__col">{props.stats ? <StatsRanked {...props} stats={stats} /> : null}</div>
                 </div>
-                <div className="statstab__aside">
-                    {props.stats ? <StatsGameModes {...props} stats={stats} /> : null}
+                <div className="statstab__row">
+                    <div className="statstab__col ranking">
+                        <StatsRankings {...props} rank={rank} />
```

Comment:
can you hide these if the player has no rank?
im the old version this was just hidden
![image](https://user-images.githubusercontent.com/8034086/42862157-9db8d704-8a5e-11e8-9779-b32758892677.png)